### DISCUSSION TITLE
I love this

### DISCUSSION THREAD

**Discussion by bogordesaincom** [mantinedev/mantine#460] [Kind words]
Thanks Mantine for help me built the my app

<img width="1426" alt="Jepretan Layar 2021-11-26 pukul 10 32 29" src="https://user-images.githubusercontent.com/51335197/143522670-15dc0511-422b-4ad3-a923-e292bce8289c.png">

**Comment by rtivital** [comment]
Thanks for kind words and sharing your app!
